platform/broadcom/sonic-platform-modules-dell/z9332f/sonic_platform/sfp.py

```diff
@@ -1178,7 +1191,7 @@ def get_tx_power(self):
                 if channel_monitor_data is not None:
                     tx1_pw = self._strip_unit_from_str(channel_monitor_data['data']['TXPower']['value'])
                 else:
-                    tx1_pw = 0.0
+                    tx1_pw = "N/A"
```

Comment:
tx_power_list should be empty if the channel_monitor_data is empty due to eeprom read failure. By appending 'N/A' the test will pass even though the eeprom read failed.  The tx_power_list should be 'N/A' iff self.qsfp_dd_txpower_supported is false.